src/index.js

```diff
@@ -1,16 +1,19 @@
 import './style.css';
 import {
-  addTask, editTask, toggleTask, removeTask,
+  addTask,
+  editTask,
+  toggleTask,
+  removeTask,
 } from './modules/crud.js';
 
 const allTasks = JSON.parse(localStorage.getItem('tasks'));
 let tasks = [];
 
 const paper = document.getElementById('paper');
 
-if (allTasks === null) {
+if (allTasks.length === 0 || allTasks === null) {
   tasks = [];
-  if (paper.childNodes.length < 4) {
+  if (paper.childNodes.length > 7) {
```

Comment:
- [ ] This part of code is throwing an error, `Cannot read properties of null (reading 'length')` This is happening on my end because my localStorage has no records so JS cannot get the length of nothing (null). You were not getting error because your local storage already had 'tasks' records.  
 Kindly address this issue, to do that you have to first check if allTasks == null, if null then initialise empty array or set an empty tasks list in local storage as soon as the program starts. 

PS. You can run `localStorage.clear()`  in web browser's console to clear the localStorage records and simulate the error 😉